lab/topologies/lab00_cisco_base/docker-compose.yml

```diff
@@ -35,6 +35,7 @@ services:
     volumes:
       - ../annet/config.yaml:/config.yaml
       - ../topologies/lab00_cisco_base/src/my_generators:/my_generators
+      - ../scripts:/home/ubuntu/scripts
     environment:
-      - NETBOX_TOKEN=${NETBOX_TOKEN}
+      - NETBOX_TOKEN=a630dcefcb191982869e7576190e79bfd569d33c
```

Comment:
Needs a comment, warning to never store tokens in codebase in prodiction environment.